### ISSUE TITLE
Determine how to handle Sierra ILL Lending patrons

### ISSUE THREAD

**Issue by CUSeif** [culibraries/folio#55]
Sierra has many patron accounts for ILL patrons (more than 100 and less than 1000.) These accounts exist to facilitate access for small libraries around the state to gain access to ILLiad. 

- [ ] @CUSeif will get sample names from ILL Lending/Sierra
- [ ] Talk with CIRC about patron groups these accounts belong to
- [ ] Talk with ILL to better understand process and options
- [ ] Investigate the ReShare roadmap, and if the solution is long or short-term

**Comment by libraryfemme** [comment]
@CUSeif  - just double checking, by "Sierra ILL patrons," do you mean other libraries only? There are obviously also CU faculty, staff, etc. in Sierra for borrowing. Pretty sure you aren't including those accounts in this issue.

**Comment by CUSeif** [comment]
@libraryfemme yes this for the libraries that have the non ezproxy access for ILLiad that Lisa generally manages.

**Comment by CUSeif** [comment]
A very quick search of what documentation EDS has, it appears that the service can provide OpenURLs which can be configured so that whatever info can be transferred to the specific ILLiad form that needs to have info provided rather than having to have the patron complete the form.
